test/indexer_unit_test.go

```diff
@@ -3,33 +3,30 @@ package test
 import (
```

Comment:
These changes are technically unrelated to code generation.

The first version of our unit tests took a specific response and verified a single field. This replaces the single-field checks with the JSON compare functions.